chain/network/src/routing.rs

```diff
@@ -257,12 +263,31 @@ pub enum FindRouteError {
 }
 
 impl RoutingTable {
-    pub fn new(peer_id: PeerId) -> Self {
+    pub fn new(peer_id: PeerId, store: Arc<Store>) -> Self {
+        // Load account peers
+        let mut account_peers = SizedCache::with_size(ANNOUNCE_ACCOUNT_CACHE_SIZE);
+        for (account_id, announce_account) in store
```

Comment:
Why we try to always fill in the cache in the beginning instead of fetching on demand?

Reply:
Rationale was, that accounts were need for sync, right now I'm only syncing with accounts stored on cache. But it makes sense that cache get updated with newer accounts. I updated this.